projects/1.20-fabric/assets/0-modrinth-mod/ubesdelight/lang/zh_cn.json

```
@@ -0,0 +1,89 @@
+{
+  "block.ubesdelight.garlic_crate": "箱装大蒜",
+  "block.ubesdelight.ginger_crate": "箱装生姜",
+  "block.ubesdelight.halo_halo_feast": "大碗杂杂混",
+  "block.ubesdelight.kalan": "烤炉",
+  "block.ubesdelight.leche_flan_feast": "菲式焦糖布丁",
+  "block.ubesdelight.lemongrass_crate": "箱装柠檬香草",
+  "block.ubesdelight.lemongrass_leaf_crop": "柠檬香草叶",
+  "block.ubesdelight.lemongrass_stalk_crop": "柠檬香草茎",
+  "block.ubesdelight.lumpia_feast": "大份菲律宾润饼",
+  "block.ubesdelight.milk_tea_ube_feast": "大碗紫薯奶茶",
+  "block.ubesdelight.ube_cake": "紫薯蛋糕",
+  "block.ubesdelight.ube_crate": "箱装紫薯",
+  "block.ubesdelight.wild_garlic": "野生大蒜",
+  "block.ubesdelight.wild_ginger": "野生生姜",
+  "block.ubesdelight.wild_lemongrass": "野生柠檬香草",
+  "block.ubesdelight.wild_ube": "野生紫薯",
+  "item.ubesdelight.arroz_caldo": "鸡肉米粥",
+  "item.ubesdelight.bangsilog": "虱目鱼鸡蛋蒜饭",
```

```suggestion
  "item.ubesdelight.bangsilog": "蒜炒饭配煎虱目鱼与煎蛋",
```
据我这边查到的：XXsilog本身由煎蛋（Itlog）、炒饭（Sinangag）和另一样“配菜”（比如`Bangus`和`Tocino`）组成，许多食谱中都将他们称为“Popular breakfasts”。所以或许这里这个应被译为“蒜炒饭配虱目鱼与煎蛋”。
~~其实还想着类比英式早餐，给它安一个“菲式XX早餐”的名字来着。因为不论是Bangsilog还是下文的Tosilog抑或其他的XXsilog，silog即煎蛋与炒饭总是标配。~~

---

我查到文章也是说蒜饭加蛋加配菜，但是想着精简（
（所以是把不该精简的地方精简了（悲